package.json

```diff
@@ -8,7 +8,9 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
+    "sass": "^1.34.0",
```

Comment:
Not this time, if you don't have a real good reason to use `styled-components` I want you to use it please ;-)